service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/history/SealedOrderHistoryService.java

```diff
@@ -82,7 +88,7 @@ public Map<String, Object> generate(CaseData caseData) {
             .childrenDescription(getChildrenForOrder(selectedChildren, caseData))
             .specialGuardians(appointedGuardianFormatter.getGuardiansNamesForTab(caseData))
             .othersNotified(othersNotifiedGenerator.getOthersNotified(selectedOthers))
-            .document(sealedPdfOrder)
+            .document(order)
```

Comment:
double check this as this isn't the sealed document anymore

Reply:
For manual_upload, we can leave the document fields blank at this about-to-submit stage, then set the documents at submitted stage.